webfront/views/entry.py

```diff
@@ -205,6 +206,10 @@ def get(
 
     @staticmethod
     def filter(queryset, level_name="", general_handler=None):
+        if level_name.lower() == "tigrfams":
+            full_path = general_handler.request.get_full_path().lower()
+            return redirect(full_path.replace("tigrfams", "NCBIfam"))
```

Comment:
Keeping URL in lower case for consistency:

```suggestion
            return redirect(full_path.replace("tigrfams", "ncbifam"))
```